exercises/series/src/test/java/SeriesTest.java

```diff
@@ -17,145 +17,122 @@
     public ExpectedException expectedException = ExpectedException.none();
 
     @Test
-    public void hasDigitsShort() {
-        Series series = new Series("01234");
-        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4);
-        List<Integer> actual = series.getDigits();
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
+    public void slicesOfOneFromOne() {
+        Series series = new Series("1");
+        List<List<String>> expected = Arrays.asList(Arrays.asList("1"));
+        List<List<String>> actual = series.slices(1);
         assertEquals(expected, actual);
     }
-
+    
     @Ignore("Remove to run test")
     @Test
-    public void hasDigitsLong() {
-        Series series = new Series("0123456789");
-        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
-        List<Integer> actual = series.getDigits();
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
-        assertEquals(expected, actual);
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void keepsTheDigitOrderIfReversed() {
-        Series series = new Series("9876543210");
-        List<Integer> expected = Arrays.asList(9, 8, 7, 6, 5, 4, 3, 2, 1, 0);
-        List<Integer> actual = series.getDigits();
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
+    public void slicesOfOneFromTwo() {
+        Series series = new Series("12");
+        List<List<String>> expected = Arrays.asList(
+                Arrays.asList("1"), 
+                Arrays.asList("2")
+        );
+        List<List<String>> actual = series.slices(1);
         assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void keepsArbitraryDigitOrder() {
-        Series series = new Series("936923468");
-        List<Integer> expected = Arrays.asList(9, 3, 6, 9, 2, 3, 4, 6, 8);
-        List<Integer> actual = series.getDigits();
+    public void slicesOfTwo() {
+        Series series = new Series("35");
+        List<List<String>> expected = Arrays.asList(Arrays.asList("3", "5"));
+        List<List<String>> actual = series.slices(2);
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
```

Comment:
As for the previous tests, the two assetions above can be removed I think. But I'm happy to open a new issue for that if you prefer 🙂 That applies for all the other tests below as well 🙂